api/files/serializers.py

```diff
@@ -188,8 +188,12 @@ def user_id(self, obj):
 
     def update(self, instance, validated_data):
         assert isinstance(instance, FileNode), 'Instance must be a FileNode'
-        for attr, value in validated_data.items():
-            setattr(instance, attr, value)
+        if len(validated_data) == 1 and 'checkout' in validated_data:
```

Comment:
1) It's not currently possible, and could only break if on a manual query to the API that tries to update multiple metadata fields, not if the request was made through [the UI (via Fangorn)](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/fangorn.js#L1037-L1052)
2) `len` validation was originally included because `'checkout'` is returned on first `.save()` call, so that would currently work, it could break if/when we migrate more functionality to APIv2